src/passes/OptimizeInstructions.cpp

```diff
@@ -982,6 +982,14 @@ struct OptimizeInstructions
         }
       }
     } else if (auto* binary = boolean->dynCast<Binary>()) {
+      if (binary->op == SubInt32) {
+        if (auto* num = binary->left->dynCast<Const>()) {
+          if (num->value.geti32() == 0) {
+            // 0 - x   ==>   x
```

Comment:
this comment looks confusing. how about `bool(0 - x)   ==>   bool(x)`?

Reply:
Sure. Done!